backend/model/model.go

```diff
@@ -1355,6 +1336,28 @@ func MigrateDB(ctx context.Context, DB *gorm.DB) (bool, error) {
 		return false, e.Wrap(err, "Error creating idx_daily_session_counts_view_project_id_date")
 	}
 
+	if err := DB.Exec(`
+		CREATE MATERIALIZED VIEW IF NOT EXISTS daily_error_counts_view AS
```

Comment:
should we only index error objects that are newer than 3 months old, similarly to how we optimized the sessions materialized view?

Reply:
good point! updated (and updated the sessions one since that change wasn't in our code)

Reply:
thanks!